### DISCUSSION TITLE
Exporting as VTK for pre-MODFLOW 6

### DISCUSSION THREAD

**Discussion by savendano-gw** [modflowpy/flopy#1931] [Q&A] [answered]
Hello,

I am having issues exporting as vtk for MF2005 and MFNWT. I was following the example from flopy3_vtk_export.py from July 13, 2021 for the mf2005 Freyberg model and then applying the same code to an mfNWT model. When I look in ViSit, I only get mesh as an option (for example when I export heads, there is no 'head' scalar option) and it throws an error when I try to plot the mesh. Any ideas what may be going on? 

Thank you!

**Comment by wpbonelli** [comment, marked as answer]
Hi @savendano-gw would you mind trying on your end with a recent flopy (latest release is 3.4.2, or you could install from the tip of the development branch), the [current version of the VTK export tutorial](https://github.com/modflowpy/flopy/blob/develop/.docs/Notebooks/export_vtk_tutorial.py), and the latest available VisIT for your platform? If you still see the issue, if you could provide your mfnwt version and OS, that may help to track it down. Maybe there is something OS-specific going on.

I tried swapping in mfnwt for mf2005 and was able to plot heads from VisIt 3.3.3 (macOS)

<img width="1253" alt="Screenshot 2023-09-11 at 12 30 46 PM" src="https://github.com/modflowpy/flopy/assets/10502868/11e08af2-f1ba-40ff-accd-1ae13fbbb89b">

**Comment by savendano-gw** [comment]
All this worked probably because I used the most current tutorial; sorry I think I was so focused on getting the MF6 part to work that I didn't realize the tutorial also already included Mf2005. Thank you!
